### ISSUE TITLE
Hello! I think it would be Great if you could make the Taskbars Colorable

### ISSUE THREAD

**Issue by Foxwyi** [dremin/RetroBar#776]
I think it would be a really Nice feature

**Comment by 111ebobo** [comment]
> I think it would be a really Nice feature

"System" ones are actually colorable if you're using the classic theme

**Comment by DanMiller299** [comment]
I agree! Especially the Vista Aero theme. Right now it looks too similar to the regular Vista Basic
